Makefile

```diff
@@ -254,7 +254,7 @@ install:
 	   cd $(INSTALL_BINDIR); \
 	   ln -sf ocamlc.byte$(EXE) ocamlc$(EXE); \
 	   ln -sf ocamllex.byte$(EXE) ocamllex$(EXE); \
-	   fi
+	fi
```

Comment:
Sorry - just looked at the raw file. GitHub (or my browser) is rendering the diff view badly - lines 253-256 are showing at the same indentation. Sorry for the noise...